ccpm/scripts/batch-prd-complete.sh

```
@@ -0,0 +1,265 @@
+resolve_prd() {
+  local input="$1"
+
+  # If it's already a full path that exists, use it
+  if [ -f ".claude/prds/${input}.md" ]; then
+    echo "$input"
+    return 0
+  fi
+
+  # If it looks like just a number, try to glob match
+  if [[ "$input" =~ ^[0-9]+$ ]]; then
+    local matches=(.claude/prds/${input}-*.md)
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)